Cargo.toml

```diff
@@ -8,6 +8,7 @@ repository = "https://github.com/linebender/druid"
 readme = "README.md"
 categories = ["os::macos-apis", "os::windows-apis", "gui"]
 edition = "2018"
+resolver = "2"
```

Comment:
We might consider updating to 2021, but that might be out of scope for this PR.

Reply:
When playing around I updated to 2021 and it "just worked".